include/effects/matrix/PatternWeather.h

```diff
@@ -153,17 +152,17 @@ class PatternWeather : public LEDStripEffect
         return false;
     }
 
-    inline float KelvinToFarenheit(float K)
+    static inline float KelvinToFarenheit(float K)

```

Comment:
Good one. In hindsight we should have done that straight away. :)

Reply:
Do we need this on the other two conversion methods in PatternWeather.h?  If so I can squeeze them into the changes i am working on now.

Reply:
@mggates39 They already are in; you can check the "Files changed" tab on the PR's page in the GitHub website. I just decided to comment on the first occurence. :)

Reply:
Sweet one less thing for me to worry about.  That's what I get for looking at it late at night and not scrolling down.

Reply:
Yeah. Been there, done that.